edx_exams/apps/lti/views.py

```diff
@@ -23,8 +28,53 @@
 from edx_exams.apps.core.statuses import ExamAttemptStatus
 from edx_exams.apps.lti.utils import get_lti_root
 
+log = logging.getLogger(__name__)
+
 EDX_OAUTH_BACKEND = 'auth_backends.backends.EdXOAuth2'
 
+LTI_PROCTORING_ASSESSMENT_CONTROL_ACTIONS = [
+    'pauseRequest',
+    'resumeRequest',
+    'terminateRequest',
+    'update',
+    'flagRequest',
+]
+
+@api_view(['POST'])
+@require_http_methods(['POST'])
+@authentication_classes((Lti1p3ApiAuthentication,))
+@permission_classes((LtiProctoringAcsPermissions,))
+def acs_endpoint(request, lti_config_id):
+    """
+    Endpoint for ACS actions
+
+    NOTE: for now just have the actions LOG what the actions is doing.
+    We can implement proper functionality and tests later once we
+    hear back from our third party proctoring service vendors
+    (i.e. Verificient and Proctortrack).
+    """
+    # TODO:
+    # First, have access_token in lti-consumer add actions to the access token
+    data = json.loads(request.body)
+    action = data['action']
+
+    if action == 'flag':
```

Comment:
I'd recommend doing a bit more than just looking at the action here. We can take the user, resource, and attempt number in this request to look up the attempt object that we would be modifying or flagging. Can we throw some errors if those parameters are incorrect? Maybe make our log message more explicit on exactly what attempt is being flagged?